app/views/orders/receipt.html.haml

```diff
@@ -47,10 +47,13 @@
         - if @order.has_subsidies?
           %td.currency= order_detail.wrapped_subsidy
         %td.currency=order_detail.wrapped_total
+  %tfoot
     %tr
       - colspan = @order.has_subsidies? ? 5 : 4
-      %td{colspan: colspan}
+      %td.hide-on-mobile{colspan: colspan}
         %strong= t('.total')
-      %td.currency= number_to_currency(@order_details.sum { |d| d.total || 0 })
+      %td.currency
+        .responsive-header Total
```

Comment:
Is it the colspan that's making this necessary? If it is necessary, can we use the locale?